src/NHibernate.Test.VisualBasic/NHibernate.Test.VisualBasic.vbproj

```diff
@@ -145,6 +145,9 @@
     <EmbeddedResource Include="Issues\NH2966\Issues.NH2966.Mappings.hbm.xml" />
     <EmbeddedResource Include="Issues\NH3302\Issues.NH3302.Mappings.hbm.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
```

Comment:
This is a VS change that should very likely be undone, as it looks already done in master. (This change is a sort of "marking as test project" [thing](http://stackoverflow.com/q/18614342/1178314) by Microsoft.)